client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { map, get, last } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/***
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+import { getPostCommentsTree } from 'state/comments/selectors';
+
+class ConversationCaterpillarComponent extends React.Component {
+	static propTypes = {
+		blogId: PropTypes.number.isRequired,
+		postId: PropTypes.number.isRequired,
+	};
+
+	render() {
+		const { commentsTree, translate } = this.props;
+		const lastComment = commentsTree[ last( Object.keys( commentsTree ) ) ];
+		const lastCommenterName = get( lastComment, 'data.author.name' );
+		const commentCount = Object.keys( commentsTree ).length;
+
+		// At the moment, we just show authors for the entire commentsTree
+		return (
+			<div className="conversation-caterpillar">
+				{ map( commentsTree, comment => {
+					return (
+						<Gravatar
+							className="conversation-caterpillar__gravatar"
+							key={ comment.data.ID }
+							user={ comment.data.author }
+							size={ 32 }
+							aria-hidden="true"
+						/>
+					);
+				} ) }
+				<button
+					className="conversation-caterpillar__count"
+					title={
+						commentCount > 1
+							? translate( 'View comments from %(commenterName)s and %(count)d others', {
+									args: {
+										commenterName: lastCommenterName,
+										count: commentCount - 1,
+									},
+								} )
+							: translate( 'View comment from %(commenterName)s', {
+									args: {
+										commenterName: lastCommenterName,
+									},
+								} )
+					}
+				>
+					{ commentCount > 1
+						? translate( '%(commenterName)s and %(count)d others', {
+								args: {
+									commenterName: lastCommenterName,
+									count: commentCount - 1,
+								},
+							} )
+						: translate( '%(commenterName)s commented', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( '%s commented: ' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).